src/test/java/com/synopsys/integration/jenkins/scan/bridge/BridgeDownloadTest.java

```diff
@@ -14,7 +14,7 @@ public class BridgeDownloadTest {
     private final BridgeDownload bridgeDownloadMock = Mockito.mock(BridgeDownload.class);
 
     @Test
-    void downloadSynopsysBridgeTest() throws PluginExceptionHandler {
+    void downloadSynopsysBridgeTest() throws Exception {
```

Comment:
Is there any specific reason for changing the exception from PluginExceptionHandler to Exception.